docs/functionality.rst

```diff
@@ -309,6 +309,37 @@ Finally, the implementation of the scheduler in the FMD
 is based on the appscheduler.schedulers.Background schedulers
 about which you can read more `in the corresponding documentation page <apscheduler.schedulers>`_.
 
+
+Adding database cleaning schedule
+----------------------------------
+
+As your application grows, so will the amount of data stored in the database.
+To prevent it from growing too large, you can set up a schedule to clean the database.
+Pruning is fully customizable, and can be set up to run as often as you want deleting data as old as you want through
+your own cron schedule.
+
+This can be done by adding the following code to your application:
+
+.. code-block:: python
+
+    weeks_to_keep = 8
+    delete_custom_graph_data = True
+    pruning_cron_schedule = {'month':"*/2",
```

Comment:
Oh I only just noticed you meant the config. That would be pretty nice - I'll change it, but I'll still argue for keeping the cron as dict instead of a string.